### ISSUE TITLE
[Azure] Object Detection, about create new compute 

### ISSUE THREAD

**Issue by LaiHT1211** [dotnet/machinelearning-modelbuilder#2775]
How to how to solve this problem: "This workspace does not have any valid compute sizes. Try a workspace in a different region."
![f6345c72a929767d9f3791c67180b9a](https://github.com/dotnet/machinelearning-modelbuilder/assets/107231008/d0921011-003b-41ab-8f35-6f49b991ff6c)

**Comment by LittleLittleCloud** [comment]
Which version of model builder are you using. We just updated the supported compute size inside model builder recently.

@zewditu  for FYI

**Comment by TheFedex87** [comment]
I'm facing the same issue. How can I solve the problem?

**Comment by zewditu** [comment]
@TheFedex87   where is the region of your workspace?

**Comment by TheFedex87** [comment]
West Europe, but I have resolved this afternoon. I download the updated extension of model builder and now the train of object detection can also be done locally without use any Azure resource. However tomorrow I will try also using Azure to check if this new version can work properly on Azure too.

**Comment by zewditu** [comment]
@TheFedex87, I hope the latest will work on latest Model Builder I am able to create new compute on West Europe region. If you still hit the issue let us know. 

Thanks!

**Comment by TheFedex87** [comment]
@zewditu I confirm that with the new updated extension I'm also able to create the compute on Azure. The error message "This workspace does not have any valid comput sizes" is gone!!
